bot/exts/recruitment/talentpool/_review.py

```diff
@@ -433,11 +433,15 @@ async def _previous_nominations_review(self, member: Member) -> Optional[str]:
 
         nomination_times = f"{num_entries} times" if num_entries > 1 else "once"
         rejection_times = f"{len(history)} times" if len(history) > 1 else "once"
+        nomination_vote_threads = ", ".join(
+            [f"<#{nomination.thread_id}>" if nomination.thread_id else '' for nomination in history]
```

Comment:
I think we need to use the jump URL here too. It's a bit annoying as it's quite long and we can't use the link syntax because it's not in an embed, but I think it should be fine.